plugins/meta/tuning/README.md

```diff
@@ -32,3 +32,32 @@ Some network sysctls are documented in the Linux sources:
 - [Documentation/sysctl/net.txt](https://www.kernel.org/doc/Documentation/sysctl/net.txt)
 - [Documentation/networking/ip-sysctl.txt](https://www.kernel.org/doc/Documentation/networking/ip-sysctl.txt)
 - [Documentation/networking/](https://www.kernel.org/doc/Documentation/networking/)
+
+## Netlink Operation
+The parameters, "mac", "mtu" and "promisc", changes the interface attributes as followings:
+
+```
+{
+  "name": "mytuning",
+  "type": "tuning",
+  "promisc": true,
+  "mac": "c2:b0:57:49:47:f1",
+  "mtu": 1454
+}
+```
+
+## Netlink configuration reference
```

Comment:
And "Interface attribute configuration reference"

Reply:
Fixed.